hstream/src/HStream/Server/Core/Query.hs

```diff
@@ -221,8 +268,10 @@ createQueryWithNamespace'
               validateNameAndThrow ResView view
               Core.createView' sc view sources sink builder persist createQueryRequestSql rSQL createQueryRequestQueryName
               >>= hstreamViewToQuery metaHandle
-            _ -> throw $ HE.WrongExecutionPlan "Create query only support create stream/view <name> as select statements"
-        _ -> throw $ HE.WrongExecutionPlan "Create query only support create stream/view <name> as select statements"
+            _ -> throw $ HE.WrongExecutionPlan "Create query only support create2 stream/view <name> as select statements"
+        _ -> throw $ HE.WrongExecutionPlan "Create query only support create3 stream/view <name> as select statements"
+        
+        
```

Comment:
The same as L215, are these your local debugging modifications? And how about the extra blank lines?